lib/.eslintrc.yaml

```diff
@@ -3,3 +3,4 @@ rules:
   require-buffer: error
   buffer-constructor: error
   no-let-in-for-declaration: error
+  alphabetize-errors: error
```

Comment:
Per @mscdex's comment It might be best to replace this with a:
```js
/* esline-enable alphabetize-errors */
```
in `lib/internal/errors.js`